### PR TITLE
Header, footer and typography

### PR DESCRIPTION
**Pull request by georgecrawford**
- [x] verify typography and buttons

### CODE REVIEW

File: scss/style.scss [hunk 1]
```diff
@@ -582,49 +527,49 @@ a:hover {
...
-}
-
-.amp-access-nav__link {
-	color: white;
-	@include oTypographySans(s);
-	text-decoration: none;
-	margin: 0 0.5em;
-	cursor: pointer;
-}
+// .amp-access-nav {
```
Comment (quarterto) [inline scss/style.scss]: These should be removed

File: views/article.html [hunk 2]
```diff
@@ -79,26 +83,69 @@
...
-					<div class="amp-access-nav" amp-access="session" amp-access-hide>
-						<ul class="amp-access-nav__links">
-							<li class="amp-access-nav__link-item">
-								<a class="amp-access-nav__link" on="tap:amp-access.login-logout">Sign out</a>
-							</li>
-							<li class="amp-access-nav__link-item">
-								<a class="amp-access-nav__link" href="http://registration.ft.com/registration/selfcare/">My Account</a>
-							</li>
-						</ul>
+			<header class="o-header">
```
Comment (quarterto) [inline views/article.html]: Do we want this in a partial as well?

File: views/article.html [hunk 3]
```diff
@@ -67,7 +71,7 @@
 		</script>
 	</head>
 	<body>
-		<amp-analytics config="//SOURCE_HOST{{SOURCE_PORT}}/analytics/config.json">
+		<amp-analytics config="//SOURCE_HOSTNAME{{SOURCE_PORT}}/analytics/config.json">
```
Comment (quarterto) [inline views/article.html]: I fixed this in the ads branch by removing `SOURCE_HOST` entirely and just using a relative path

Comment (georgecrawford) [inline views/article.html]: We discussed this, and I'm not sure whether a relative path is supported on CDNs (https://www.ampproject.org/docs/reference/extended/amp-analytics.html doesn't make this clear). So I'd prefer to keep it as absolute.

File: views/article.html (hunk 2 above)
Comment (georgecrawford) [inline views/article.html]: Fixed

File: scss/style.scss (hunk 1 above)
Comment (georgecrawford) [inline scss/style.scss]: Fixed, thanks

**Pull request merged**